controllers/customers/customer-controller.js

```diff
@@ -38,4 +38,38 @@ const sendMessage = async (id, message) => {
   }
 }
 
-module.exports = { getCustomers, sendMessage }
\ No newline at end of file
+// Sned Message to Multiple Customers
+const sendMessageToMultipleCustomers = async (message, selectedCustomers) => {
```

Comment:
After what we talked today `sendMessageToSingleCustomer`, and `sendMessageToMultipleCustomers` will have to live in the text message controller. This refactoring will also require you to first create the text message in the DB. So then you can send the text message to the customers.

I also see there is some duplication in the code for `sendMessageToSingleCustomer`, and `sendMessageToMultipleCustomers`. Check [this article](https://codinglead.co/javascript/what-is-DRY-code) about DRY code.
A way we can refactor this to not have duplication can be the following:
- Move this code to the text message controller.
- Ensure `sendMessage` always receives the same arguments for example `sendMessage = async (messageId, customers)`. Customers is going to be an array of customers, and that value can be either 0, 1, ...
- then you can iterate over the list of customers, and call `sendTwilioSms` per customer.